test/automated_generation_e2e_test.ts

```
@@ -4,6 +4,8 @@
 import * as fs from "fs/promises";
 import * as path from "path";
 import type * as monaco from "monaco-editor";
+import { ElectronAPI } from "@/types/electron";
```

ESLint

[error] Remove this unused import of 'ElectronAPI'.

(sonarjs/unused-import)